### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by magicDGS**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at <htsjdk-next-beta@email.com>. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (magicDGS) [inline CODE_OF_CONDUCT.md]: @tfenne, @jacarey, @lbergelson - I just create the placeholder for the email (`htsjdk-next-beta@email.com`) but we should decide which contact we will use. Maybe we can add here a mailing list...

Comment (lbergelson) [reply CODE_OF_CONDUCT.md]: Yeah, we need an email address, we could make a google group?  I would probably go with htsjdk-next-maintainers@(somewhere.something)

Comment (magicDGS) [reply CODE_OF_CONDUCT.md]: So is it decided that it would be called `htsjdk-next` or is it only a temporary name/email?

Comment (magicDGS) [reply CODE_OF_CONDUCT.md]: @lbergelson - which one is the email for the google group to include here?

**Comment by lbergelson** [comment]
@magicDGS Currently it's contact@htjsdk.org.  We can change it if we want.  Did you receive the test email I sent to it?

**Comment by magicDGS** [comment]
@lbergelson - I got the email and changed the email on the PR
@jacarey / @tfenne - are you also ok with merging this?

**Comment by lbergelson** [comment]
👍

**Pull request closed without merging**